scylla_post_install.sh

```diff
@@ -26,17 +30,42 @@ fi
 
 AMB_SUPPORT=`grep -c ^CapAmb: /proc/self/status`
 
+KERNEL_VER=$(uname -r)
+
 # AmbientCapabilities supported from v229 but it backported to v219-33 on RHEL7
 if [ $SYSTEMD_VER -ge 229 ] || [[ $SYSTEMD_VER -eq 219 && $SYSTEMD_REL -ge 33 ]]; then
     if [ $AMB_SUPPORT -eq 1 ]; then
+        AMB_CAPABILITIES="CAP_SYS_NICE CAP_IPC_LOCK"
+        # CAP_PERFMON is only available on linux-5.8+
+        if version_ge $KERNEL_VER 5.8; then
+            AMB_CAPABILITIES="$AMB_CAPABILITIES CAP_PERFMON"
+        fi
         mkdir -p /etc/systemd/system/scylla-server.service.d/
         cat << EOS > /etc/systemd/system/scylla-server.service.d/capabilities.conf
 [Service]
-AmbientCapabilities=CAP_SYS_NICE CAP_IPC_LOCK
+AmbientCapabilities=$AMB_CAPABILITIES
 EOS
     fi
 fi
 
+DEBIAN=$(echo $ID $ID_LIKE | grep -oi debian)
+if ! version_ge $KERNEL_VER 5.8; then
+    # On older kernel environment, we have to relax perf_event_paranoid setting
+    # since there is no CAP_PERFMON.
+    PERF_EVENT_PARANOID=1
+elif [ "$DEBIAN" ]; then
```

Comment:
instead of hardwiring to a certain distro, how about:
```sh
elif [ $(cat /proc/sys/kernel/perf_event_paranoid) -eq 4 ]; then
    # On Debian/Ubuntu, it deny access from non-root even with CAP_PERFMON
    # It requires to set perf_event_paranoid=2 to use CAP_PERFMON with non-root
    PERF_EVENT_PARANOID=2
fi
```

Reply:
s/-eq/-ge/ for future compatibility

Reply:
Replaced to:
```
elif [ $(cat /proc/sys/kernel/perf_event_paranoid) -ge 3  ]; then
```

NOTE: Why `-ge 3` is, because Debian uses perf_event_paranoid=3 for root-only mode, and Ubuntu uses perf_event_paranoid=4 for same feature, we need to handle both.